File: 4th Semester/Design and programming environments/Project Java Client-Server/Model/src/main/java/domain/Participant.java
package domain;

public class Participant extends Entity<Integer>{
    private String nume;
    private int varsta;

    public Participant(String nume, int varsta) {
        this.nume = nume;
        this.varsta = varsta;
    }

    public Participant() {

    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    public int getVarsta() {
        return varsta;
    }

    public void setVarsta(int varsta) {
        this.varsta = varsta;
    }

    @Override
    public String toString() {
        return "Participant{" +
                "nume='" + nume + '\'' +
                ", varsta=" + varsta +
                '}';
    }
}

File: 4th Semester/Design and programming environments/Proiect C-Sharp Client-Server Sockets/AppModel/Participant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AppModel
{
    public class Participant : Entity<int>
    {
        public Participant(int v, string n)
        {
            varsta = v;
            nume = n;
        }
        public int varsta { get; set; }

        public string nume { get; set; }

        public override string ToString()
        {
            return nume + " " + varsta;
        }
    }
}
